php/php.editor/src/org/netbeans/modules/php/editor/actions/UsedNamesCollector.java

```diff
@@ -143,10 +106,14 @@ private boolean existsUseForTypeName(final Collection<? extends UseScope> declar
 
     private static class NamespaceNameVisitor extends DefaultVisitor {
         private final OffsetRange offsetRange;
-        private final Map<String, List<UsedNamespaceName>> existingNames = new HashMap<>();
+        private final NamespaceScope namespaceScope;
+        private NamespaceScope currentScope;
```

Comment:
Nitpick: we usually put `final` fields before the non-final ones.

Reply:
Fixed.